docker/buildDocker.sh

```diff
@@ -1,13 +1,30 @@
 #!/bin/bash
 set -u
 
-jdkVersion=''
-bootDir=''
-buildVariant="hotspot"
+BOOTDIR=''
```

Comment:
single quotes used here and double quotes on the next line - should we be consistent here?

Reply:
Fixed with latest commit